src/hooks/useToast.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+const Ctx = React.createContext(null);
+
+const ToastContainer = props => (
+  <div style={{ position: "fixed", right: 0, top: 0 }} {...props} />
+);
+const Toast = ({ children, onDismiss }) => (
```

Comment:
Could we move this component into a separate `components` folder? I'm not really sure about how we structure the architecture but I think it would be better to leave the hooks to only do logical stuff